components/CovidMap.js

```diff
@@ -0,0 +1,702 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tabletop from 'tabletop';
+import MarkerClusterer from '@google/markerclustererplus';
+
+const mapStyle = {
+  height: '400px', 
+  width: '100%',
+};
+
+const clusterStyleFirstSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleFirstMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleFirstLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleSecondSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleSecondMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleSecondLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleThirdSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-third'
+}
+
+const formStyle = {
+  width: '50%',
+  float: 'left', 
+}
+
+const legendStyle = {
+  width: '50%', 
+  float: 'right',
+}
+
+const legendItemStyle = {
+  width: '100%',
+  marginBottom: '5px',
+}
+
+const legendTextStyle = {
+  width: '90%',
+  height: '20px',
+  float: 'left',
+  textAlign: 'right',
+}
+
+const legendIconStyle = {
+  minWidth: '30px',
+  height: '20px',
+  float: 'right',
+  display: 'inline-block',
+}
+
+const legendIconFirst = {
+  background: '#0B5D93',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'right',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+const legendIconSecond = {
+  background: '#634562',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+const legendIconThird = {
+  backgroundColor: '#CE2727',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+
+class Map extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      map: null,
+      firstClusterer: null,
+      secondClusterer: null, 
+      thirdClusterer: null,
+      fetchedMap: false,
+      data: [],
+      selectUpdate: true,
+      selectedOption: props.value || 'all',
```

Comment:
It doesn't look like we pass in props to this component. What do you think about dumping this reference?

```suggestion
      selectedOption: 'all',
```

Reply:
Yes.